Updates Computer Setup Instructions

giovannadf opened the pull request:
Updated computer setup instructions with new images and re-ordering of steps. Also fixes a typo on another page.

giovannadf committed:
Updates Computer Setup Instructions

Updated computer setup instructions with new images and re-ordering of steps. Also fixes a typo on another page.

rchlfryn reviewed (approved):
Looks good to me (LGTM) - looks like you have one conflict. LMK if you need help resolving it before merging it

giovannadf committed:
Merge branch 'gh-pages' into Computer-Setup-Edits

giovannadf committed:
Adds suggested changes

Revision of the first pull request. Addes suggested changes.

giovannadf committed:
Computer Setup - Updates

Updates screenshots and edits copy on instructions. Includes suggested changes and edits from CS team.
